practice-one/src/styles/components/introduction.css

```diff
@@ -52,6 +52,49 @@
   font-weight: 400;
 }
 
+
```

Comment:
remove redundant empty line, recheck for all

Reply:
i removed it thanks